client/state/reader/thumbnails/actions.js

```diff
@@ -45,33 +45,37 @@ export const requestThumbnail = ( embedUrl ) => ( dispatch ) => {
 		case 'videopress': {
 			const posterEndpoint = `https://public-api.wordpress.com/rest/v1.1/videos/${ id }/poster`;
 
-			return globalThis.fetch( posterEndpoint ).then( async ( response ) => {
-				let json;
-				try {
-					json = await response.json();
-				} catch ( error ) {}
+			try {
+				return globalThis.fetch( posterEndpoint ).then( async ( response ) => {
```

Comment:
I'm not a fan of mixing `.then` with `async/await`. I think we should settle on one or the other. Could be a neat style improvement :)

Reply:
Agreed. In https://github.com/Automattic/wp-calypso/pull/79111/commits/b0115c336f6bad6293de163dd8908fcc4cf69c0e I've updated it to use `.then()` to keep the same pattern of returning the promise.